libr/core/cmd_search.c

```diff
@@ -1317,7 +1317,7 @@ static int r_core_search_rop(RCore *core, RInterval search_itv, int opt, const c
 			result = false;
 			goto bad;
 		}
-		map->fd = core->io->desc->fd;
+		map->fd = r_io_fd_get_current (core->io);
```

Comment:
about this kind of NULL deref... my question is: why is there a NULL there?

Reply:
because if you open no file there's no current desc